core/templates/components/oppia-angular-root.component.ts

```diff
@@ -62,8 +62,8 @@
  */
 
 // We are using eslint disable here for multilines because we have exceeded
-// max-len limit at a lot of places so it is not possible to use eslint disable
-// next line for each of them.
+// max-len limit at a lot of places so it is not possible to use "eslint disable
```

Comment:
> The exclusion only works if the import is on a single line. Also, in this file, there are also cases where max-line property is violated and is not an import statement.

Then please put all the imports in a single line each, and have a per-line eslint check for the non-import lines which violate (or try to find some way to break the line if doing so makes sense).

Reply:
Done.